unified_targets/configs/VGOODRCF4.config

```diff
@@ -0,0 +1,124 @@
+# version
+# Betaflight / VGOODRCF4 (VGF4) 4.1.0 Apr 28 2019 / 09:06:34 () MSP API: 1.42
+
+# start the command batch
+batch start

```

Comment:
Can you please remove the `batch start` and `batch end` commands.

And then rebase / squash into one commit please.